### PR TITLE
LGA-2799: Announce we’re calling from CLA

### PR DESCRIPTION
**Pull request by BenMillar-CGI**
## What does this pull request do?
- Adds a new NullBooleanField to the PersonalDetails model determining whether the call centre operator can announce if they are calling from Civil Legal Advice.

## Any other changes that would benefit highlighting?

- Adds migrations to add the new field to the database
- Adds 'announce_call' as a field to the personal details unit tests, defaulting to `False`

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by TawneeOwl — APPROVED** [review]
LGTM

File: cla_backend/apps/legalaid/models.py [hunk 1]
```diff
@@ -158,6 +158,8 @@ class PersonalDetails(CloneModelMixin, TimeStampedModel):
     # only normalised version of post code for now
     search_field = models.TextField(null=True, blank=True, db_index=True)
 
+    announce_call = models.BooleanField(blank=False, default=False)
```
Comment (said-moj) [inline cla_backend/apps/legalaid/models.py:161]: @BenMillar-CGI Should this be a null boolean field? So that old records can have a default value of null

Comment (BenMillar-CGI) [reply cla_backend/apps/legalaid/models.py:161]: Hi @said-moj, this implementation defaults all existing records to False, which was the requested behaviour. However, the frontend will display the warning whether the value is null or False. 

I am happy to update this if you think this is the best way to approach it 👍

Comment (BenMillar-CGI) [reply cla_backend/apps/legalaid/models.py]: Hi @said-moj I have updated this model to use `NullBooleanField` as suggested. Many thanks!! 😄

**Review by TawneeOwl — APPROVED** [review]
LGTM

**Review by digitalali-moj — APPROVED** [review]
LGTM!

**Pull request merged**